src/appendices/contributors.md

```diff
@@ -181,6 +181,7 @@ If you contributed to the BIDS ecosystem and your name is not listed, please add
 | Ilona Lipp                                           | 📖                                     |
 | International Neuroinformatics Coordinating Facility | 💵📋                                   |
 | Iris Groen                                           | 📖                                     |
+| Isaac To                                             | 📖                                     |
```

Comment:
remove from here, add to correct wiki address: see
- https://github.com/bids-standard/bids-specification/pull/2072

Reply:
done